generated/nitclk/session.py

```diff
@@ -163,6 +163,9 @@ def __init__(self, session_number, encoding='windows-1251'):
         self._session_number = session_number
```

Comment:
Based on that logic, all files should start with '_'. None of them are imported directly by the customer. They are all imported for them by `__init__.py`.

It will be somewhat complicated to import different named files in `__init__.py` if they are different. On the other hand, we are already adding conditional logic there.

Reply:
> based on that logic, all files should start with '_'. 

Yeah, it's inconsistent that `_library.py` and `_attributes.py` start with `_` but `session.py` does not. I think it's just a historical inconsistency that would confuse new developers. Do you agree we should open issue one way or another?